### PR TITLE
UIU-1707: Search index username

### PR DESCRIPTION
**Pull request by rchr**
https://issues.folio.org/browse/UIU-1707

This PR adds a dropdown to UserSearch to enable searching on single fields explicitly, in this case the field username.
To enable that the PR adds searchableIndexes to UserSearch's SearchField.

### CODE REVIEW

File: translations/ui-users/en.json [hunk 1]
```diff
@@ -813,5 +813,8 @@
   "bulkClaimReturned.status.notOk": "Item could not be claimed returned: item is already claimed returned",
   "bulkClaimReturned.items.ok": "{numItems} item(s) <strong>successfully</strong> claimed returned.",
   "bulkClaimReturned.items.notOk": "{numItems} item(s) <strong>not</strong> claimed returned.",
-  "bulkClaimReturned.moreInfoPlaceholder": "Enter more information about the claim (required)"
+  "bulkClaimReturned.moreInfoPlaceholder": "Enter more information about the claim (required)",
+
+  "index.username": "Username",
+  "index.all": "All"
```
Comment (zburke) [inline translations/ui-users/en.json:819]: The mockup indicates `Keyword (name, email, identifier)` rather than `All`.

**Review by zburke — CHANGES_REQUESTED** [review]
The code looks great, but please correct the `index.all` translation value.

File: translations/ui-users/en.json (hunk 1 above)
Comment (rchr) [reply translations/ui-users/en.json]: You are right! I changed that accordingly to the mockup.

**Review by zburke — APPROVED** [review]
LGTM! Please squash and merge. 

Thank you!

**Pull request merged**